sources/packages/web/src/services/AuthService.ts

```diff
@@ -93,6 +107,20 @@ export class AuthService {
       });
 
       if (this.keycloak.authenticated) {
+        // In case of user closed browser without logout.
+        window.addEventListener("beforeunload", this.logUserClosedBrowser);
+        if (AuditService.hasUserClosedBrowser()) {
+          // In case of user reopened browser with an active session.
+          await ApiClient.AuditApi.audit({ event: AuditEvent.BrowserReopened });
+          AuditService.resetUserClosedBrowser();
+        }
```

Comment:
I would say that `AuditService.hasUserClosedBrowser()` and `if (AuditService.wasUserLoginTriggered())` are mutually exclusive and an `else` can be added.